.github/workflows/smoke-test-daily.yml

```diff
@@ -32,6 +32,8 @@ jobs:
 
             - name: Setup WooCommerce Monorepo
               uses: ./.github/actions/setup-woocommerce-monorepo
+              with:
+                install: '@woocommerce/plugin-woocommerce...'
```

Comment:
By default the monorepo setup action won't install or build anything. This lets us use it to get platform dependencies and caches set up without forcing it to do any work. It accepts `true` for a normal `pnpm install`, `false` (the default), and a `--filter` input so that different consuming workflows can limit the scope of installs and builds to maximize performance.

Since there are other plugins with teams actively developing them in the monorepo I don't think we should default to installing dependencies for WooCommerce Core. We _could_ default to `true` instead of `false` but, to be honest, I kind of like the explicit nature of an action like this. It makes sure that folks are actively opting-in to whatever performance penalties are associated with the specific action they're taking.